src/library/basesqltablemodel.cpp

```diff
@@ -293,7 +324,7 @@ void BaseSqlTableModel::select() {
     // should not disturb that if we are only removing tracks.
     qStableSort(rowInfo.begin(), rowInfo.end());
 
-    m_trackIdToRows.clear();
+    TrackId2Rows trackIdToRows;
```

Comment:
Same comment from the previous PR -- this was a change we made for performance reasons in ~2010.

How big is your library? Do select() times look different before and after this change for you?

Since every developer's library differs, we could really benefit from some standard benchmarks for decision making.